### DISCUSSION TITLE
How many thirdparty library did you used?

### DISCUSSION THREAD

**Discussion by emma-eva** [vlang/v#15739] [Installation Issues]
Hi,
V wont work cli mode without `garbage collector `, so need to install libgc library. Like that how many library used in V & where? Also, will it possible to release full fill self compiler? After compiling using self compile feature still i can see tcc compiler data on compiled V file. When will be release `source to source` compiler without using thirdparty compiler?

Waiting for your good answer :)

**Comment by JalonSolov** [comment]
Check the `thirdparty` directory for which libs are used.

V will be fully working without other compilers when the `-native` option is complete.

**Reply by emma-eva** [reply]
when will be it complete? how many days/months will take?

**Reply by JalonSolov** [reply]
We don't do dates.  It will be done when it is done.  There is an open PR now for another improvement.

**Comment by spytheman** [comment]
> When will be release source to source compiler without using thirdparty compiler?

If you just want a `.c` file from a `.v` file (source to source), you can just do:

`v -o file.c file.v` - it will produce the C source, and then stop, without invoking the C compiler for you.
